modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
I don't understand how it is worst for testing. It might be easier to test here by passing graph, but then how do you test DI outside fx? There may be more work involved in setting up DI for test, but it will at least be transparent. 
The graph is a dump of object in either case, but right now we are exposing it to test passing it around. IMHO dig users should not be aware of the graph.